packages/components/src/components/DaScroll.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    class="scroll__switch"
+    :class="[showed ? 'shown' : '']"
+    @click="scrollTop"
+  >
+    <svgicon name="arrow" class="scroll__switch__icon" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "DaScroll",
+
+  data() {
+    return {
+      showed: false,
+    };
+  },
+
+  methods: {
+    scrollTop() {
+      window.scrollTo({ top: 0, behavior: "smooth" });
+    },
+  },
+
+  created() {
+    this.scroll = (event) => {
+      this.showed =
+        document.documentElement.scrollTop >
+        document.documentElement.scrollHeight / 3;
```

Comment:
I think it should be `/ 2` to make it 50%, right?
```suggestion
        document.documentElement.scrollHeight / 2;
```

Reply:
Your welcome, first how run service on windows or can use online service posts? second for 50% scroll when text devide by 3 is better!

Reply:
You can check out the instructions here:
https://github.com/dailydotdev/daily#-running-dailydev-locally

Reply:
when run daily-apps_daily-gateway_1 and daily-apps_daily-api_1 have no such file or directory error!

Reply:
Let's discuss a separate issue. Please open a new issue and provide all the details.